GitExtensions/Properties/launchSettings.json

```diff
@@ -6,6 +6,10 @@
     },
     "Dashboard": {
       "commandName": "Project"
+    },
+    "Blame": {
```

Comment:
Generate all configurations?

Reply:
This is just a another configuration for running GE with different arguments in the debugger:

![grafik](https://user-images.githubusercontent.com/36601201/200958127-510b6669-8dae-43b2-ae8a-f6ce3f5056ef.png)

Reply:
I meant if all possible ways to GE should be added.
The standalone blame is a form the probably should be removed.